tutorials/export/android_gradle_build.rst

```diff
@@ -57,3 +57,20 @@ configuration is needed.
 
     For example, ``_example/image.png`` will **not** be included as an asset,
     but ``_image.png`` will.
+
+Disable Swipe-to-Dismiss on Wear OS
+-----------------------------------
+
+To disable swipe-to-dismiss functionality on Wear OS devices, follow these steps:
+
+1. Open the file ``res://android/build/res/values/themes.xml`` in your file editor.
+2. Add or update the ``android:windowSwipeToDismiss`` attribute to ``false`` in the `GodotAppMainTheme` style.
+
+Example:
+
+.. code-block:: xml
+
+    <style name="GodotAppMainTheme" parent="@android:style/Theme.DeviceDefault.NoActionBar">
+        <item name="android:windowDrawsSystemBarBackgrounds">false</item>
+        <item name="android:windowSwipeToDismiss">false</item>
```

Comment:
@syntaxerror247 We could add an advanced export option to automatically update this value for gradle builds.

Reply:
@m4gr3d I initially considered this approach but decided to document instructions instead, as it seemed feasible only for Gradle export. However, after seeing your other comment, if it's acceptable to have Gradle-specific build options, I can close this PR and open a new one in the Godot repository to implement it.

Do you think adding this to the project settings in `pointing` would be a better solution?